pegasus/sites.v3/code.org/public/teacher-dashboard/index.haml

```diff
@@ -589,34 +592,32 @@ title: <%= I18n.t('dashboard_landing_title').inspect %>
 
   }]);
 
-  if (showSectionProjects) {
-    app.controller('SectionProjectsController', ['$scope', '$routeParams', 'sectionsService',
-                                               function($scope, $routeParams,  sectionsService) {
-      $scope.sections = sectionsService.query();
-      $scope.section = sectionsService.get({id: $routeParams.id});
-      $scope.tab = 'projects';
-
-      // Angular does not offer a reliable way to wait for the template to load,
-      // so do it using a custom event here. The call to listen for the custom
-      // event must not be nested inside another deferred call or we might
-      // miss the event.
-      $scope.$on('section-projects-rendered', function () {
-        $scope.section.$promise.then(
-          function (section) {
-            window.renderSectionProjects(section.id);
-          }
-        );
-      });
-
-      // the ng-select in the nav compares by reference not by value, so we can't just set
-      // selectedSection to section, we have to find it in sections.
-      $scope.sections.$promise.then(
-        function (sections) {
-          $scope.selectedSection = $.grep(sections, function(section) { return (section.id == $routeParams.id);})[0];
+  app.controller('SectionProjectsController', ['$scope', '$routeParams', 'sectionsService',
+      function($scope, $routeParams,  sectionsService) {
+    $scope.sections = sectionsService.query();
+    $scope.section = sectionsService.get({id: $routeParams.id});
+    $scope.tab = 'projects';
+
+    // Angular does not offer a reliable way to wait for the template to load,
+    // so do it using a custom event here. The call to listen for the custom
+    // event must not be nested inside another deferred call or we might
+    // miss the event.
+    $scope.$on('section-projects-rendered', function () {
+      $scope.section.$promise.then(
+        function (section) {
+          window.renderSectionProjects(section.id);
         }
       );
-    }]);
-  }
+    });
+
+    // the ng-select in the nav compares by reference not by value, so we can't just set
+    // selectedSection to section, we have to find it in sections.
+    $scope.sections.$promise.then(
+      function (sections) {
+        $scope.selectedSection = $.grep(sections, function(section) { return (section.id == $routeParams.id);})[0];
```

Comment:
nit: i believe our style guide says this function should be on multiple lines. also, is the == vs. === intentional?